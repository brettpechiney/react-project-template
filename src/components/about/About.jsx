import React from 'react';
import { hot } from 'react-hot-loader/root';
import styled from 'styled-components';

const About = () => (
  <Wrapper>
    <p className="intro">This is the about page.</p>
  </Wrapper>
);

const Wrapper = styled.section`
  display: flex;
  align-items: center;
  justify-content: center;

  .intro {
    font-size: large;
  }
`;

export default hot(About);
