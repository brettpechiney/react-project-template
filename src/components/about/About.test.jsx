import React from 'react';
import { shallow } from 'enzyme';
import About from './About';

const setup = propOverrides => {
  const props = Object.assign({}, propOverrides);

  const wrapper = shallow(<About {...props} />);

  return {
    props,
    wrapper,
  };
};

describe('<About />', () => {
  const { wrapper } = setup();

  it('renders', () => {
    expect(wrapper.exists()).toBe(true);
  });
});
