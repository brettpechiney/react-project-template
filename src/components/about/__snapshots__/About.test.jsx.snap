// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> renders correctly 1`] = `
<section
  className="About__Wrapper-r6nm9a-0 fvjtwO"
>
  <p
    className="intro"
  >
    This is the about page.
  </p>
</section>
`;
